packages/hawtio/src/plugins/camel/routes-service.test.ts

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { render, screen, cleanup } from '@testing-library/react'
+import { jolokiaService } from '@hawtiosrc/plugins/connect/jolokia-service'
+import { routesService } from './routes-service'
+import { MBeanNode } from '@hawtiosrc/plugins/shared/tree'
+import fs from 'fs'
+import path from 'path'
+import $ from 'jquery'
+import { IconNames } from './icons'
+
+jest.mock('@hawtiosrc/plugins/connect/jolokia-service')
+
+describe('routes-service', () => {
+  const owner = 'test-tree'
+  let contextNode: MBeanNode
+  let routesNode: MBeanNode
+  let simpleRouteNode: MBeanNode
+
+  const testRouteId = 'simple'
+  const routesXmlPath = path.resolve(__dirname, 'testdata', 'camel-sample-app-routes.xml')
+  const sampleRoutesXml = fs.readFileSync(routesXmlPath, { encoding: 'utf8', flag: 'r' })
+  const doc: XMLDocument = $.parseXML(sampleRoutesXml as string)
+  const simpleRouteXml = $(doc).find("route[id='" + testRouteId + "']")[0]
+
+  jolokiaService.execute = jest.fn(
+    async (mbean: string, operation: string, args?: unknown[] | undefined): Promise<unknown> => {
```

Comment:
in `args?: unknown[] | undefined`, no need for `| undefined` since `args` is assigned with `?`.